File: src/lib/scala/Slate.Core/src/main/scala/slate/core/mods/ModService.scala
/**
  * <slate_header>
  * author: Kishore Reddy
  * url: https://github.com/kishorereddy/scala-slate
  * copyright: 2016 Kishore Reddy
  * license: https://github.com/kishorereddy/scala-slate/blob/master/LICENSE.md
  * desc: a scala micro-framework
  * usage: Please refer to license on github for more info.
  * </slate_header>
  */
package slate.core.mods

import slate.entities.core.{EntityRepo, EntityService}
import slate.common.results.{ResultSupportIn}

class ModService(repo:EntityRepo[Mod]) extends EntityService[Mod](repo) with ResultSupportIn {

}

File: src/support/integration/src/main/kotlin/kiit/integration/mods/ModService.kt
/**
 * <slate_header>
 * url: www.slatekit.com
 * git: www.github.com/code-helix/slatekit
 * org: www.codehelix.co
 * author: Kishore Reddy
 * copyright: 2016 CodeHelix Solutions Inc.
 * license: refer to website and/or github
 * about: A tool-kit, utility library and server-backend
 * mantra: Simplicity above all else
 * </slate_header>
 */

package kiit.integration.mods

import kiit.entities.Entities
import kiit.entities.EntityRepo
import kiit.entities.EntityService

class ModService(entities: Entities, repo: EntityRepo<Long, Mod>) : EntityService<Long, Mod>(repo)